Makefile.Common

```diff
@@ -42,7 +42,10 @@ IMPI=impi
 BUILD_TYPE?=release
 
 # COVER_PKGS is the list of packages to include in the test coverage
-COVER_PKGS := $(shell go list ./... | tr "\n" ",")
+EXCLUDE_MODS=-not -path "./examples/*" -not -path "./tests/*"
```

Comment:
```suggestion
EXCLUDE_MODS=-not -path "./examples/*"
```
Although coverage for `./tests` isn't super helpful, it will be helpful to show if there's test helper code that isn't actually used (and can be deleted).